src/components/controls/km-input-lstring-ml.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div :id="id" class="km-input-lstring-ml mb-2">
+    {{ binding }}
+    <div v-for="(item, index) in binding" :key="item">
+      <div class="row">
+        <div class="col-md-11">
+          <km-input-lstring
+            v-model="item.value"
+            :locales="props.locales"
+            v-on:update:modelValue="emitChange"
+          ></km-input-lstring>
+        </div>
+        <div class="col-md-1">
+          <button
+            :disabled="binding.length === 1"
+            class="btn btn-danger btn-sm d-flex align-items-center justify-content-center"
+            type="button"
+            @click="removeItem(item, index)"
+          >
+            <i class="bi bi-trash"></i> {{ t("remove") }}
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <button
+      class="btn btn-outline-secondary btn-sm d-flex align-items-center justify-content-center"
+      type="button"
+      @click="addItem()"
+      :disabled="hasEmpty"
+    >
+      <i class="bi bi-plus"></i> {{ t("add") }}
+    </button>
+  </div>
+</template>
+
+<script setup>
+import {
+  ref,
+  computed,
+  onMounted,
+  defineProps,
+  defineEmits,
+  useAttrs,
+} from "vue";
+import { isEmpty } from "lodash";
+import KmInputLstring from "./km-input-lstring.vue";
+import { removeEmpty } from "../../utils/helpers.js";
+
+function t(str) {
+  return str;
+}
+const props = defineProps({
+  locales: {
+    type: Array,
+    required: true,
```

Comment:
put a validation that the lcoales are less than <=3 chars

Reply:
added validation for locales with check for <= 3 characters